recipes-kernel/linux/files/template_overlay.dts

```diff
@@ -0,0 +1,6 @@
+/dts-v1/;
```

Comment:
Originally added to facilitate adding new overlays in case we plan on writing any. My plan was to separate overlays used to enable i2c buses and other overlays (such as ones to alter ethernet, change SD/MMC pins) but seems like it may be less relevant now.

Reply:
Oh I see. I'm fine if you prefer to keep it, just make sure there is comment to describe the file usage then I'm good.